db/migrate/20181031100150_add_migrated_to_module_api_cartos.rb

```diff
@@ -0,0 +1,5 @@
+class AddMigratedToModuleAPICartos < ActiveRecord::Migration[5.2]
+  def change
+    add_column :module_api_cartos, :migrated, :boolean
```

Comment:
default à false ?

Reply:
et contrainte not null ?

Reply:
C'est un flag temporaire et le default: false peux bloquer la base sur les grosses tables.